x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_utility_bar/translations.ts

```diff
@@ -42,6 +42,13 @@ export const ADDITIONAL_FILTERS_ACTIONS_SHOW_BUILDING_BLOCK = i18n.translate(
   }
 );
 
+export const ADDITIONAL_FILTERS_ACTIONS_SHOW_THREAT_MATCHES_ONLY = i18n.translate(
+  'xpack.securitySolution.detectionEngine.alerts.utilityBar.additionalFiltersActions.showThreatMatchesOnly',
+  {
+    defaultMessage: 'Show threat indicator matches only',
```

Comment:
```suggestion
    defaultMessage: 'Show only threat match alerts',
```

Thoughts: I would emphasize the entities that this filter is acting on (alerts), and move "only" to the beginning of the label since it's logically different than the existing building block query. 

RE qualifying alert, I like "threat indicator alert" more than "threat indicator match alert" as that seems redundant, but we should probably rope product in for the final say here. /cc @MikePaquette @devonakerr

Reply:
I think "threat indicator match alert" is awkward, and "threat indicator alert" is more concise without losing meaning.

Reply:
updating as `threat indicator alerts`